requirements.txt

```diff
@@ -1,17 +0,0 @@
-# Install the current directory as an editable package. Denpendencies required
-# for installing the package and running unit tests are defined in setup.py.
--e .
-
-# Nice for development.
```

Comment:
Could be nice to add these under a `dev` target for install, but I'm not really sure how the tests, etc. have all that setup anymore.

Reply:
Lint tools are in pre-commit.yaml and test deps are in tox.ini. I'd imagine substantial development to the library to be made in another "host" Django project, with this library installed as `-e`ditable, so that host project would have the dev tools.

Reply:
The test project is specifically setup so you can run it locally, which is why you need to be able to install these requirements and then run `python manage.py runserver`.

Reply:
See https://github.com/valohai/django-allauth-2fa/blob/main/tests/run_settings.py

Reply:
Gotcha.

Reply:
Renamed to `requirements-dev.txt` instead.